core/src/main/kotlin/api/RunsRoute.kt

```diff
@@ -282,24 +282,25 @@ fun Route.runs() = route("runs") {
                     val advisorJobInFinishedState = jobs?.advisor?.status in JobStatus.SUCCESSFUL_STATUSES
                     val evaluatorJobInFinishedState = jobs?.evaluator?.status in JobStatus.SUCCESSFUL_STATUSES
 
-                    val issuesCount = if (analyzerJobInFinalState) issueService.countForOrtRunId(ortRun.id) else null
+                    val issuesCount =
+                        if (analyzerJobInFinalState) issueService.countForOrtRunIds(ortRun.id) else null
 
                     val packagesCount =
-                        if (analyzerJobInFinishedState) packageService.countForOrtRunId(ortRun.id) else null
+                        if (analyzerJobInFinishedState) packageService.countForOrtRunIds(ortRun.id) else null
```

Comment:
Nit: I haven't checked, but maybe some of these now still fit a single line again, which would be good to keep in order to reduce the diff. Similar below.

Reply:
Yeah, I checked the other ones before but looks like I missed the issuesCount